examples/multiple_servo_pdo_tuning.py

```diff
@@ -0,0 +1,285 @@
+import time
+from dataclasses import dataclass
+from typing import Union
+
+import matplotlib.pyplot as plt
+from ingenialink.pdo import RPDOMapItem, TPDOMapItem
+from ingeniamotion.enums import FilterType, OperationMode
+from ingeniamotion.motion_controller import MotionController
+
+SRV_1: str = "1"
+SRV_2: str = "2"
+
+PDO_REFRESH_RATE: float = 0.001  # seconds
+CAPTURE_TIME: int = 10  # seconds
+
+SQUARE_WAVE_AMPLITUDE_A: float = 5
```

Comment:
max and min?

Reply:
The square wave alternates between levels A and B, without defined maximum or minimum values.